resolwe/test/testcases/__init__.py

```diff
@@ -61,8 +59,17 @@ def _clean_up(self):
 
     def setUp(self):
         """Prepare environment for test."""
+        from resolwe.elastic.builder import index_builder
+
         super().setUp()
 
+        # Reset Elastic search indices.
+        index_builder.destroy()
+        index_builder.discover_indexes()
+        index_builder.create_mappings()
+        index_builder.unregister_signals()
```

Comment:
Do you need to call `unregister_signals `? It is already called as a part of `destroy()`.

Reply:
I like to be on the safe side and I think this operation is quite fast.